source/wp-content/themes/wporg-parent-2021/patterns/heading-with-arrow-and-button.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: Heading with arrow and button
+ * Slug: wporg-parent-2021/heading-with-arrow-and-button
+ * Categories: header
+ */
+
+?>
+<!-- wp:group {"align":"full","backgroundColor":"blueberry-1"} -->
+<div class="wp-block-group alignfull has-blueberry-1-background-color has-background">
+	<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"97px","bottom":"97px","right":"80px","left":"80px"}}},"className":"heading-with-arrow","layout":{"type":"flex","orientation":"vertical","justifyContent":"center","flexWrap":"nowrap"}} -->
+	<div class="wp-block-group alignfull heading-with-arrow" style="padding-top:97px;padding-right:80px;padding-bottom:97px;padding-left:80px">
+		<!-- wp:group {"style":{"spacing":{"blockGap":"0px"}},"layout":{"inherit":false}} -->
+		<div class="wp-block-group">
+			<!-- wp:group {"style":{"spacing":{"padding":{"bottom":"67px"}}},"className":"heading-with-arrow__row","layout":{"type":"flex","flexWrap":"nowrap","orientation":"horizontal","justifyContent":"center","verticalAlignment":"top"}} -->
+			<div class="wp-block-group heading-with-arrow__row" style="padding-bottom:67px">
+				<!-- wp:group {"layout":{"contentSize":"760px","inherit":false}} -->
+				<div class="wp-block-group">
+					<!-- wp:paragraph {"style":{"typography":{"fontSize":"120px","fontStyle":"normal","fontWeight":"300","lineHeight":"1.1","letterSpacing":"-0.02em"}},"textColor":"white"} -->
+					<p class="has-white-color has-text-color" style="font-size:120px;font-style:normal;font-weight:300;letter-spacing:-0.02em;line-height:1.1">We'll help you get started</p>
```

Comment:
This should be a heading block (you can default to `h2`), and the text inside it should be a link.